scaleway/resource_lb_test.go

```diff
@@ -479,10 +479,11 @@ func TestAccScalewayLbLb_WithPrivateNetworksOnDHCPConfig(t *testing.T) {
 					type        = "DEV1-S"
 					image       = "debian_bullseye"
 					enable_ipv6 = false
-				
-					private_network {
```

Comment:
Please check that waiting process is still working? may we just use the wait from private_nic now